includes/discovery/sensors/state/f5.inc.php

```diff
@@ -100,6 +100,88 @@
 } // End if (is_array($temp1))
 
 
+// Real Sync State:
```

Comment:
Can you remove all of the MIB comments please, we can get that info from the MIB itself so no need to have it here.